.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+## What type of issue would you like to report?
+
+- [ ] Bug
+- [ ] Feature request
+- [ ] Other, please explain:
+
+## Check if your issue is already reported
```

Comment:
This information is still redundant, we want to minimize this template as much as possible. The best thing is to put this information in http://docs.ckeditor.com/#!/guide/dev_issues_readme and link it here simply as a one liner like "Check if your issue is already reported".